### ISSUE TITLE
Code sample can not work with browserify.

### ISSUE THREAD

**Issue by 9n** [cytoscape/cytoscape.js-cose-bilkent#10]
the code snippet from README can not work with browserify:
```
var cytoscape = require('cytoscape');
var regCose = require('cytoscape-cose-bilkent');
regCose( cytoscape ); //register extension
```

Instead, code snippet blow works:
```
var cytoscape = window.cytoscape = require('cytoscape');
require('cytoscape-cose-bilkent');
```

**Comment by ugurdogrusoz** [comment]
I am not sure what this is. Please let us know if there is something we can do to help.

**Comment by maxkfranz** [comment]
Published as 1.2.1

Browserify seems to mangle manual UMD definitions, so I switched it to let browserify do it automatically
